core/consensus/babe/impl/block_executor.cpp

```diff
@@ -296,8 +296,8 @@ namespace kagome::consensus {
       if (res.has_error()
           && res
                  != outcome::failure(
-                     transaction_pool::TransactionPoolError::TX_NOT_FOUND)) {
-        return res;
+                        transaction_pool::TransactionPoolError::TX_NOT_FOUND)) {
+        return res.error();
```

Comment:
AIR ```as_failure()``` should be used? But I'm not sure.

Reply:
.error() returns an std:error_code, so I believe it should be fine.